.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2.1
+
+executors:
+  python:
+    docker:
+      - image: cimg/python:3.11.2
+    resource_class: small
+
+filter-not-main: &filter-not-main
+  filters:
+    branches:
+      ignore:
+        - main
+        - /^dependabot/(?!docker/).*/
+        - /^hotfix\/.+/
+
+filter-main: &filter-main
+  filters:
+    branches:
+      only:
+        - main
+        - /^hotfix\/.+/
+
+orbs:
+  serverless-framework: circleci/serverless-framework@2.0.1
+  slack: circleci/slack@4.3.0
+
+jobs:
+  check:
+    executor: python
+    steps:
+      - checkout
+      - run: pip install ruff
+      - run: make check
+  # test:
```

Comment:
My plan is to plumb together integration and then unit tests, here, at some point